tools/config/manage-dependencies.ts

```diff
@@ -0,0 +1,1163 @@
+    'wasi',
+    'worker_threads',
+    'zlib',
+  ];
+
+  if (!options.fix) {
+    reportProgress('Checking imports of Node.js APIs');
+
+    let errorsFound = false;
+    Object.values(libraries).forEach((lib) => {
+      lib.tsImports = Object.values(lib.tsImports)
+        .filter((imp) => {
+          if (nodeAPIs.includes(imp.importPath)) {
+            // Don't run the check in fix mode
+            if (!options.fix) {
+              // Don't allow to use node api outside of schematics spec files
+              if (
+                imp.usageIn.spec ||
+                imp.usageIn.lib ||
+                imp.usageIn.schematics
+              ) {
+                imp.files.forEach((file) => {
+                  // Allow to use Node APIs in SSR
+                  if (!file.includes('ssr')) {
+                    errorsFound = true;
+                    error(
+                      file,
+                      [
+                        `Node.js API \`${chalk.bold(
+                          imp.importPath
+                        )}\` is referenced.`,
+                      ],
+                      [
+                        `Node.js APIs can only be used in SSR code or in schematics specs.`,
+                        `You might have wanted to import it from some library instead.`,
+                      ]
+                    );
+                  }
+                });
+              }
+            }
+            return false;
+          }
+          return true;
+        })
+        .reduce((acc, curr) => {
+          acc[curr.importPath] = curr;
+          return acc;
+        }, {});
+    });
+    if (!errorsFound) {
+      success();
+    }
+  }
+}
+
+/**
+ * Filer native Node.js APIs
+ */
+function filterLocalAbsolutePathFiles(
+  libraries: Record<string, LibraryWithDependencies>,
+  options: ProgramOptions
+): void {
+  if (!options.fix) {
+    reportProgress('Checking absolute path imports');
+
+    let errorsFound = false;
+    Object.values(libraries).forEach((lib) => {
+      lib.tsImports = Object.values(lib.tsImports)
+        .filter((imp) => {
+          if (
+            fs.existsSync(imp.importPath) ||
+            fs.existsSync(`${imp.importPath}.ts`)
+          ) {
+            // Don't run in fix mode
+            if (!options.fix) {
+              // Allow to use absolute paths for spec files
+              if (
+                imp.usageIn.lib ||
+                imp.usageIn.schematics ||
+                imp.usageIn.schematicsSpec
+              ) {
+                imp.files.forEach((file) => {
+                  errorsFound = true;
+                  error(
+                    file,
+                    [
+                      `Absolute import should not be used outside of spec files.\n    Referenced \`${chalk.bold(
+                        imp.importPath
+                      )}\``,
+                    ],
+                    [
+                      `You should use absolute paths only for testing modules.`,
```

Comment:
Maybe we phrase it like this:
`You should use absolute paths only in test files`?